rustls/src/record_layer.rs

```diff
@@ -150,6 +152,12 @@ impl RecordLayer {
         self.write_seq >= SEQ_HARD_LIMIT
     }
 
+    /// Return true if we have ever decrypted a message. This is used in place
```

Comment:
Ordering: I would typically move trivial getters like this to the bottom of the `impl`.